[2022-08-06 19:36:59,159] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [queued]>
[2022-08-06 19:36:59,174] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [queued]>
[2022-08-06 19:36:59,174] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:36:59,174] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:36:59,174] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:36:59,190] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-11-02 00:00:00+00:00
[2022-08-06 19:36:59,196] {standard_task_runner.py:52} INFO - Started process 991 to run task
[2022-08-06 19:36:59,199] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-11-02T00:00:00+00:00', '--job-id', '287', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpygmzog7r', '--error-file', '/tmp/tmp6817907b']
[2022-08-06 19:36:59,199] {standard_task_runner.py:80} INFO - Job 287: Subtask second_task
[2022-08-06 19:36:59,277] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:36:59,379] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-02T00:00:00+00:00
[2022-08-06 19:36:59,380] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:36:59,381] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:36:59,394] {subprocess.py:85} INFO - Output:
[2022-08-06 19:36:59,395] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:36:59,396] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:36:59,433] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211102T000000, start_date=20220806T193659, end_date=20220806T193659
[2022-08-06 19:36:59,490] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:36:59,547] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:50:53,071] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [queued]>
[2022-08-06 21:50:53,087] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [queued]>
[2022-08-06 21:50:53,087] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:50:53,087] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:50:53,087] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:50:53,104] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-11-02 00:00:00+00:00
[2022-08-06 21:50:53,112] {standard_task_runner.py:52} INFO - Started process 1192 to run task
[2022-08-06 21:50:53,116] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-11-02T00:00:00+00:00', '--job-id', '291', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpvps_kuqf', '--error-file', '/tmp/tmpzi0dh35b']
[2022-08-06 21:50:53,116] {standard_task_runner.py:80} INFO - Job 291: Subtask second_task
[2022-08-06 21:50:53,246] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:50:53,351] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-02T00:00:00+00:00
[2022-08-06 21:50:53,352] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:50:53,353] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:50:53,366] {subprocess.py:85} INFO - Output:
[2022-08-06 21:50:53,368] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:50:53,368] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:50:53,404] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211102T000000, start_date=20220806T215053, end_date=20220806T215053
[2022-08-06 21:50:53,448] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:50:53,493] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:56:19,049] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [queued]>
[2022-08-06 21:56:19,062] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [queued]>
[2022-08-06 21:56:19,063] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:19,063] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:56:19,063] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:19,080] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-11-02 00:00:00+00:00
[2022-08-06 21:56:19,087] {standard_task_runner.py:52} INFO - Started process 984 to run task
[2022-08-06 21:56:19,090] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-11-02T00:00:00+00:00', '--job-id', '291', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmptqlkigsg', '--error-file', '/tmp/tmpz4pbhfpr']
[2022-08-06 21:56:19,091] {standard_task_runner.py:80} INFO - Job 291: Subtask second_task
[2022-08-06 21:56:19,171] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:56:19,261] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-02T00:00:00+00:00
[2022-08-06 21:56:19,262] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:56:19,263] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:56:19,278] {subprocess.py:85} INFO - Output:
[2022-08-06 21:56:19,280] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:56:19,281] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:56:19,315] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211102T000000, start_date=20220806T215619, end_date=20220806T215619
[2022-08-06 21:56:19,342] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:56:19,378] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:11:21,641] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [queued]>
[2022-08-06 22:11:21,656] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [queued]>
[2022-08-06 22:11:21,656] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:21,656] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:11:21,656] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:21,672] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-11-02 00:00:00+00:00
[2022-08-06 22:11:21,678] {standard_task_runner.py:52} INFO - Started process 1002 to run task
[2022-08-06 22:11:21,682] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-11-02T00:00:00+00:00', '--job-id', '294', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp9uoc7l0w', '--error-file', '/tmp/tmpu9vwmdkx']
[2022-08-06 22:11:21,682] {standard_task_runner.py:80} INFO - Job 294: Subtask second_task
[2022-08-06 22:11:21,771] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:11:21,873] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-02T00:00:00+00:00
[2022-08-06 22:11:21,874] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:11:21,874] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:11:21,890] {subprocess.py:85} INFO - Output:
[2022-08-06 22:11:21,892] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:11:21,893] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:11:21,923] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211102T000000, start_date=20220806T221121, end_date=20220806T221121
[2022-08-06 22:11:21,975] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:11:22,023] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:19:06,195] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [queued]>
[2022-08-06 22:19:06,209] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [queued]>
[2022-08-06 22:19:06,209] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:19:06,209] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:19:06,209] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:19:06,223] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-11-02 00:00:00+00:00
[2022-08-06 22:19:06,232] {standard_task_runner.py:52} INFO - Started process 1008 to run task
[2022-08-06 22:19:06,236] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-11-02T00:00:00+00:00', '--job-id', '292', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpvqt5fcnz', '--error-file', '/tmp/tmpruqe1me0']
[2022-08-06 22:19:06,236] {standard_task_runner.py:80} INFO - Job 292: Subtask second_task
[2022-08-06 22:19:06,331] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [running]> on host ed53bde1c44a
[2022-08-06 22:19:06,434] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-02T00:00:00+00:00
[2022-08-06 22:19:06,435] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:19:06,435] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:19:06,453] {subprocess.py:85} INFO - Output:
[2022-08-06 22:19:06,456] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:19:06,456] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:19:06,523] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211102T000000, start_date=20220806T221906, end_date=20220806T221906
[2022-08-06 22:19:06,570] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:19:06,643] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:22:42,944] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [queued]>
[2022-08-06 22:22:42,964] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [queued]>
[2022-08-06 22:22:42,964] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:22:42,964] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:22:42,965] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:22:42,991] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-11-02 00:00:00+00:00
[2022-08-06 22:22:43,020] {standard_task_runner.py:52} INFO - Started process 1049 to run task
[2022-08-06 22:22:43,023] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-11-02T00:00:00+00:00', '--job-id', '292', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp3dza1uv4', '--error-file', '/tmp/tmplgju2nhq']
[2022-08-06 22:22:43,024] {standard_task_runner.py:80} INFO - Job 292: Subtask second_task
[2022-08-06 22:22:43,107] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-02T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:22:43,199] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-02T00:00:00+00:00
[2022-08-06 22:22:43,200] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:22:43,200] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:22:43,217] {subprocess.py:85} INFO - Output:
[2022-08-06 22:22:43,219] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:22:43,220] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:22:43,258] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211102T000000, start_date=20220806T222242, end_date=20220806T222243
[2022-08-06 22:22:43,315] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:22:43,352] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
